### PR TITLE
Create a header component  #19

### LINKED ISSUE #19: Create a Header Component
When we display the dashboard of a user we want to also display their Name, Surname, Cohort and Role. So we will need to create a header component to go within the dashboard component.

It's also a good idea to create some mock data to utilise within this component. Having the mock data set up in the same format as the wiki ensures that the real live data we send through later will be even more seamless to integrate.

_Think about whether this is likely to be functional or class based._ 

You may also need to ensure that this component has a client-side route set up too. 

Tasks:
- [x] Check the [wiki](https://github.com/kahikatea-2019/MOA/wiki) for relevant information.
- [x] Create component file. 
- [x] Create mock data which is in the same shape as the wiki and used within the component
- [x] Write tests for this component.

You won't need to do any design just yet, just the basics to get this showing.

_Ensure that the component is utilising the mock data the same way it will when it is fully connected - i.e. from an object (to later be replaced by `props`), rather than just hardcoded into the `render` method._

### PR DESCRIPTION
**Pull request by Eteroa123**
image

## Creating Header Component 

For this task I was asked to create a header component. This header component has a job that shows the students ...
         1. Name
         2.  Surname 
         3. Cohort and year 
         4. Create client-side route for component.

## What I have done

Check the wiki for relevant information.
Create component file.
Create mock data which is in the same shape as the wiki and used within the component
Write tests for this component. And also added the client-side route for component.

closes #19

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from 'react'
+import Header from './Header'
```
Comment (emilyparkes) [inline client/components/App.jsx:3]: This will be imported into Dashboard please remove changes to app.

File: client/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+class Header extends React.Component{
+render(){
+    return(
+        <div className = 'Header'>
+        <h1><p>Name</p></h1>
```
Comment (emilyparkes) [inline client/components/Header.jsx]: Please mock all data as you would expect to get it from state. 
e.g  {state.firstName}

File: client/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <h2><p>Surname</p></h2>
+        <h3><p>Harakeke2019</p></h3>       
+        </div>
+    )
+}
+}
+
+
+
+
```
Comment (emilyparkes) [inline client/components/Header.jsx]: Why so many extra lines? 😜

File: tests/client/components/Header.test.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import {shallow} from 'enzyme'
+import Header from '../../../client/components/Header'
+
+test('<Header/>', () => {
```
Comment (emilyparkes) [inline tests/client/components/Header.test.js:5]: Ill check you test after other changes are made.

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: client/components/App.jsx (hunk 1 above)
Comment (Eteroa123) [reply client/components/App.jsx:3]: Thanks sis removed through the latest commit called : "https://github.com/kahikatea-2019/MOA/pull/32/commits/7485c631f07a22df4f1706a350b1ad85dc0b3f17 "

File: client/components/Header.jsx (hunk 2 above)
Comment (Eteroa123) [reply client/components/Header.jsx]: Hi Sis thanks heaps this has been fixed with this commit : https://github.com/kahikatea-2019/MOA/pull/32/commits/9d08c82172833b4f51ba6480270709418fa1d84c

File: client/components/Header.jsx (hunk 3 above)
Comment (Eteroa123) [reply client/components/Header.jsx]: Omg im so extra !!!!!!! this has been resolved through this commit : https://github.com/kahikatea-2019/MOA/pull/32/commits/07598a4f394b4bb296ffb68d678d6edcddd4ee3b

**Comment by Eteroa123** [comment]
@emilyparkes  Kia ora sis I resolved the problems and added the new parts of my tickets which is the client side routing.

File: package.json [hunk 5]
```diff
@@ -19,7 +19,7 @@
     "db:migrate": "npm run knex migrate:latest",
     "db:seed": "npm run knex seed:run",
     "postinstall": "run-s db:migrate db:seed",
-    "test": "jest --watchAll"
+    "test": "jest -o --coverage"
```
Comment (emilyparkes) [inline package.json]: ooh take this out and you're good!

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (Eteroa123) [reply package.json]: @emilyparkes  hi sis all done from this commit  https://github.com/kahikatea-2019/MOA/pull/32/commits/b5f2d38f1cece24ebaf4a576677d021de90ef86d

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**